plugins/blame.yml

```diff
@@ -0,0 +1,18 @@
+plugin:
```

Comment:
As of v0.30.x this is now plural ie `plugins:`

Reply:
Updated it to plugins.